### ISSUE TITLE
Multiple issues with notification area icon under MATE

### ISSUE THREAD

**Issue by cmadamsgit** [pithos/pithos#260]
Several behaviors of the notification area icon (at least under the MATE desktop) changed in a negative way with Pithos 1.1.2:
- left-click on the icon used to toggle the window being visible or not, now it just brings up the same context menu as right-click
- hovering on the icon used to show the current song info, now does nothing
- using the mouse scroll wheel on the icon used to adjust the application volume up or down (depending on scroll direction); now it adjusts the system volume instead and it always goes up (no matter the scroll direction)

This is all on Fedora 23. Installing Pithos 1.1.1 instead brings back the old behavior, so it doesn't appear to be directly tied to OS/desktop environment changes.

**Comment by TingPing** [comment]
Yes, 1.1.2 now uses the AppIndicator library if it is installed. This changes the left click action to middle click which is fine IMO. The tool-tip change is not something I thought of or realized was even a feature. As for scroll wheel nothing changed on Pithos end as far as system volume but the scrolling up only must be a bug in appindicator.

A hacky temporary solution to this is to change this section to be False: https://github.com/pithos/pithos/blob/master/pithos/plugins/notification_icon.py#L22-L27

**Comment by cmadamsgit** [comment]
Setting that to False does switch all three behaviors back to the same as 1.1.1.
